docs/api.rst

```diff
@@ -25,5 +25,12 @@
 API Docs
 ========
 
+Flask extension
+---------------
 .. automodule:: invenio_mail.ext
    :members:
+
+Tasks
+---------
+.. automodule:: invenio_mail.tasks
```

Comment:
it looks like that the documentation for the tasks is not included (only the title of the page).
Maybe it's a problem of celery? (e.g. also the [Celery tasks](https://pythonhosted.org/invenio-indexer/api.html#module-invenio_indexer.tasks) page of invenio-indexer is empty)